azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/EventHubPartitionPump.java

```diff
@@ -211,7 +211,6 @@ public void onReceive(Iterable<EventData> events)
 		@Override
 		public void onError(Throwable error)
```

Comment:
If the pump calls the error handler, it shuts down, but I'm not sure what it takes to get the pump to declare an error.  While trying to test the changes I made to address this, I experimented with forcibly closing the TCP connection and the client recovered at a lower level. The pump never saw the error.